### PR TITLE
Add protocol definition for oregon scientific THN129 temp only sensor.

### PR DESCRIPTION
**Pull request by sbsrouteur**
Added THN129 sensor. Sensor follows the v2.1 protocol, and only has 1 temp measurement.

### CODE REVIEW

File: src/devices/oregon_scientific.c [hunk 1]
```diff
@@ -352,6 +353,23 @@ static int oregon_scientific_v2_1_parser(bitbuffer_t *bitbuffer) {
...
+        float temp_c = get_os_temperature(msg, sensor_id);
+        data = data_make(
+            "time",          "",            DATA_STRING, time_str,
+            "brand",         "",            DATA_STRING, "OS",
+            "model",         "",            DATA_STRING, "THN129",
+            "id",            "House Code",  DATA_INT,    get_os_rollingcode(msg, sensor_id),
+            "channel",       "Channel",     DATA_INT,    get_os_channel(msg, sensor_id), // 1 to 5
+            "battery",       "Battery",     DATA_STRING, get_os_battery(msg, sensor_id) ? "LOW" : "OK",
+            "temperature_C",  "Celcius",    DATA_FORMAT, "%.02f C", DATA_DOUBLE, temp_c,
+            "temperature_F",  "Fahrenheit", DATA_FORMAT, "%.02f F", DATA_DOUBLE, ((temp_c*9)/5)+32,
```
Comment (merbanan) [inline src/devices/oregon_scientific.c]: Remove this. Only the signal native representation can be there.

**Comment by merbanan** [comment]
Some signal samples would be nice also.

File: src/devices/oregon_scientific.c (hunk 1 above)
Comment (sbsrouteur) [inline src/devices/oregon_scientific.c]: no sure to understand what you mean, the sensor transmits, chan & bat status (in message begin after Id) then temp in degree_C. Which os these lines should I remove?

Comment (merbanan) [inline src/devices/oregon_scientific.c]: temperature_F

**Comment by sbsrouteur** [comment]
Added a sample signal file.
https://github.com/merbanan/rtl_433_tests/pull/136

**Comment by merbanan** [comment]
Thanks

**Pull request merged**